test/turnlanes.cc

```diff
@@ -127,52 +127,52 @@ TEST(Turnlanes, validate_turn_lanes) {
   // Test right active
   test_turn_lanes({VALHALLA_SOURCE_DIR "test/pinpoints/turn_lanes/right_active_pinpoint.pbf"},
                   expected_routes_size, expected_legs_size, expected_maneuvers_size, maneuver_index,
-                  "[ left | through | through;right ACTIVE ]");
+                  "[ left | through | through;*right* ACTIVE ]");
 
   // Test left active
   test_turn_lanes({VALHALLA_SOURCE_DIR "test/pinpoints/turn_lanes/left_active_pinpoint.pbf"},
                   expected_routes_size, expected_legs_size, expected_maneuvers_size, maneuver_index,
-                  "[ left ACTIVE | through | through;right ]");
+                  "[ *left* ACTIVE | through | through;right ]");
 
   // Test right most slight left active
   test_turn_lanes({VALHALLA_SOURCE_DIR
                    "test/pinpoints/turn_lanes/right_most_slight_left_active_pinpoint.pbf"},
                   expected_routes_size, expected_legs_size, expected_maneuvers_size, maneuver_index,
-                  "[ slight_left | slight_left ACTIVE | slight_right | right ]");
+                  "[ *slight_left* VALID | *slight_left* ACTIVE | slight_right | right ]");
 
   // Test slight right active
   test_turn_lanes({VALHALLA_SOURCE_DIR "test/pinpoints/turn_lanes/slight_right_active_pinpoint.pbf"},
                   expected_routes_size, expected_legs_size, expected_maneuvers_size, maneuver_index,
-                  "[ slight_left | slight_left | slight_right ACTIVE | right ]");
+                  "[ slight_left | slight_left | *slight_right* ACTIVE | right ]");
 
   // Test left most left u-turn active
   test_turn_lanes({VALHALLA_SOURCE_DIR
                    "test/pinpoints/turn_lanes/left_most_left_uturn_active_pinpoint.pbf"},
                   expected_routes_size, expected_legs_size, expected_maneuvers_size, maneuver_index,
-                  "[ left ACTIVE | left | left | through | through;right ]");
+                  "[ *left* ACTIVE | *left* VALID | *left* VALID | through | through;right ]");
```

Comment:
@mandeepsandhu with a left u-tun only one lane can be active/valid - the other left turn lanes can not be valid

Reply:
same for a right uturn